src/bstackTests/e2e.spec.js

```diff
@@ -0,0 +1,40 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../../fixtures');
+
+test.describe("End to End Tests", ()=>{
+  test('End to End test', async ({ page }) => {
+    if(process.env.BROWSERSTACK_LOCAL == true){
+      await page.goto("http://localhost:3000/");
+  }
+  else{
+      await page.goto("https://bstackdemo.com/");
+  }
```

Comment:
suggestion: can we `baseURL` inside the test configuration instead of doing this manually?
Ref: https://playwright.dev/docs/test-configuration#basic-options